layouts/index.sectionsatom.xml

```diff
@@ -55,11 +55,10 @@
             {{ with $author.copyright -}}
                 <rights>{{ . | plainify }}</rights>
             {{- end }}
-            {{- partial "utils/summary.html" $page -}}
-            {{- $summary := .Description | default ($page.Scratch.Get "summary") -}}
-            {{ printf `<summary type="html"><![CDATA[%s]]></summary>` $summary | safeHTML }}
+            {{- $summary := .Description | default (partial "utils/summary.html" $page) -}}
+            <summary type="html">{{ $summary | html }}</summary>
             {{ if $.Site.Params.includeContent }}
-                {{ printf `<content type="html"><![CDATA[%s]]></content>` .Content | safeHTML }}
+                <content type="html">{{ .Content | html }}</content>
```

Comment:
It's essentially a reverse of `safeHTML` - it removes the "safe" attribute from a string, so that it will be escaped if necessary. That's what Hugo's default `rss.xml` is using. Using `escapeHTML` here might lead to double escaping.

Reply:
Understood.